shared/util_linux.go

```diff
@@ -136,7 +141,9 @@ func LookupUUIDByBlockDevPath(diskDevice string) (string, error) {
 	return uuid[lastSlash+1:], nil
 }
 
-// Detect whether err is an errno.
+// GetErrno detects whether the error is an errno.
+//
+//revive:disable:error-return Error is returned first because this is similar to assertion.
 func GetErrno(err error) (errno error, iserrno bool) {
```

Comment:
I've disabled revive `error-return` rule using comment. The rule does not support specific exception in configuration, but I also didn't want to entirely disable it.

Rule docs: https://revive.run/r#error-return